tools/cli/dev-bundle.js

```diff
@@ -73,11 +73,19 @@ function getDevBundleForRelease(release) {
   var track = parts[0];
   var version = parts.slice(1).join("@");
 
-  var packageMetadataDir = find(
-    rootDir,
-    makeStatTest("isDirectory"),
-    ".meteor", "package-metadata"
-  );
+  var packageMetadataDir;
+  if (process.env.METEOR_WAREHOUSE_METADATA_DIR) {
+    packageMetadataDir = find(
```

Comment:
This probably shouldn't even perform the search, and directly use `METEOR_WAREHOUSE_METADATA_DIR` as the path, exactly as specified (i.e. it should not even call `find`).

Reply:
Fixed